torch/onnx/verification.py

```diff
@@ -113,14 +113,29 @@ def _ort_session(
     return ort_session
 
 
-def _compare_ort_pytorch_outputs(ort_outs, pt_outs, rtol, atol):
+def _compare_ort_pytorch_outputs(
+    ort_outs, pt_outs, rtol, atol, acceptable_error_percentage: Optional[float]
+):
```

Comment:
It'd be helpful to define and explain `acceptable_error_percentage` in the docstring here.

Reply:
done